src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/SimpleAudioPlayer.kt

```diff
@@ -20,30 +19,14 @@ import java.util.concurrent.TimeUnit
 // Named "Simple" since just displays a progress bar and play/pause button
 // No waveform view
 class SimpleAudioPlayer(private val audioFile: File, private val player: IAudioPlayer) : HBox() {
-    val playPauseButton = Button()
     val progressBar = JFXProgressBar()
-    var playGraphic: Node? = null
-        set (value) {
-            field = value
-            if (!isPlaying) {
-                playPauseButton.graphic = field
-            }
-        }
-    var pauseGraphic: Node? = null
-        set (value) {
-            field = value
-            if (isPlaying) {
-                playPauseButton.graphic = field
-            }
-        }
 
-    @Volatile private var isPlaying = false
+    @Volatile  var isPlaying = SimpleBooleanProperty(false)
```

Comment:
I believe that volatile just makes the reference volatile, not the object; and the reference isn't changing. Like; you can also probably make this a val, because whether you set it to true or false, it's not going to be pointing to a different SimpleBooleanProperty

Really, you're wanting to know if properties are thread safe (I don't know if they are)